appwidgets/package.json

```diff
@@ -32,7 +32,8 @@
   },
   "dependencies": {
     "@jupyter-widgets/base": "^1.1.10 || ^2.0.0 || ^3.0.0 || ^4.0.0",
-    "@types/jquery": "^3.5.8"
+    "@types/jquery": "^3.5.8",
+    "yarn": "^1.22.18"
```

Comment:
I did not put this into `package.json` because as far as I know `yarn` should be installed globally. But maybe it is cleaner to add it here as well. Should we move it to `devDependencies` though? And we need to change https://github.com/JanLahmann/Qoffee-Maker/tree/main/appwidgets#build-frontend

Reply:
Ok habe noch eine kleine Info ins README geschrieben wie man das global installieren kann, falls es fehlt. 

Also habe es jetzt nicht in die devDependencies gepackt. Oder soll ich das noch machen (bzw. wie geht das)?